github/view/GitApplication.kt

```diff
@@ -0,0 +1,198 @@
+                Column(modifier = Modifier.fillMaxHeight(), horizontalAlignment = Alignment.CenterHorizontally,
+                    verticalArrangement = Arrangement.Center) {
+                    TextField(value = repoTextFieldValue, singleLine = true, onValueChange = {
+                        repoTextFieldValue = it
+                        exploreButtonTextValue = "Explore $it"
+                    })
+                    Button(onClick = {
+                        coroutineScope.launch {
+                            val path = GlobalState.downloader.download(repoTextFieldValue)
+                            GlobalState.migrator.migrate(path)
+                        }
+                    }) {
+                        Text(exploreButtonTextValue)
+                    }
+                    Text(repoStatusValue())
+                }
+                Divider(color = Color.Black, modifier = Modifier.fillMaxHeight().width(1.dp))
+                Column(verticalArrangement = Arrangement.spacedBy(20.dp),
+                    horizontalAlignment = Alignment.CenterHorizontally) {
+
+                    Row(horizontalArrangement = Arrangement.spacedBy(20.dp)) {
+                        queryFieldAndButton("Which users worked on this file?") { input ->
+                            GlobalState.explorer.usersCollaboratedOnFile(
+                                input
+                            )
+                        }
+
+                        queryFieldAndButton("Which files did this user change?") { input ->
+                            GlobalState.explorer.filesEditedByUser(
+                                input
+                            )
+                        }
+
+                        queryFieldAndButton("Who worked on this repo?") { input ->
+                            GlobalState.explorer.usersWorkedOnRepo(
+                                input
+                            )
+                        }
+                    }
+                    Row(horizontalArrangement = Arrangement.spacedBy(20.dp)) {
+                        queryFieldAndButton("Who also worked on the files in this commit?") { input ->
+                            GlobalState.explorer.commitFilesAlsoWorkedOnByUsers(
+                                input
+                            )
+                        }
+                        queryFieldAndButton("How many times has this file been edited?") { input ->
+                            GlobalState.explorer.fileEditCount(
+                                input
+                            )
+                        }
+                    }
+                    OutlinedTextField(outputTextFieldValue, { }, readOnly = true, modifier = Modifier.fillMaxSize(),
+                        textStyle = TextStyle(fontFamily = FontFamily.Monospace))
+                }
+            }
+        }
+    }
+
+    @Composable
+    private fun queryFieldAndButton(description: String, explorerFn: (String) -> ArrayList<String>) {
```

Comment:
Composables use capitals by convention (unlike all other functions!)
```suggestion
    private fun QueryFieldAndButton(description: String, explorerFn: (String) -> ArrayList<String>) {
```